### PR TITLE
Improvement needs on hover of "Ok" button of source code modal on product page

### LINKED ISSUE #11216: Improvement needs on hover of "Ok" button of source code modal on product page
Improvement needs on hover of Ok button of source code modal on product page

**To Reproduce**
1. BO > Product Page > Summary/Description Tab
2. Click on 'Sorce code icon'
3. Try to hover on "Ok" button. Background-color changes to transparent which should not be.

To fix it, We need to apply below style.
`.mce-primary.mce-btn:focus, .mce-primary.mce-btn:hover {
    background-color: #005fb3 !important;
}`

**Screenshots**
hover-issue

**Additionnal information**
PrestaShop version: 1.7.4.3
PHP version: 7.1

### PR DESCRIPTION
**Pull request by dineshbadrukhiya**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Improvement needs on hover of "Ok" button of source code modal on product page
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #11216 
| How to test?  | Hover the "Ok" button of source code modal on product page

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11424)

### CODE REVIEW

**Comment by matks** [comment]
Thank you for your contribution 😄 looks good to me but since CSS is not my best I'll wait for one of my peers' review too

**Comment by dineshbadrukhiya** [comment]
@matks As you have suggested I will commit with compiled theme.css 😀

File: admin-dev/themes/new-theme/scss/components/_tinymce.sass [hunk 1]
```diff
@@ -548,14 +548,14 @@ body .mce-abs-layout-item
...
-.mce-primary
+.mce-primary.mce-btn
     min-width: 50px
     color: #fff
     border: 1px solid #b1b1b1
     border-color: rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.1) rgba(0, 0, 0, 0.25) rgba(0, 0, 0, 0.25)
     background-color: #006dcc
     &:hover, &:focus
-        background-color: #005fb3
+        background-color: #005fb3 !important
```
Comment (Quetzacoalt91) [inline admin-dev/themes/new-theme/scss/components/_tinymce.sass]: Hmm, I totally dislike the `!important`, it prevents people to update the value with another css file.

In my opinion the change to apply must be done elsewhere.

**Review by Quetzacoalt91 — CHANGES_REQUESTED** [review]
After a short investigation, it seems we can get rid of another `!important` style definition:

https://github.com/PrestaShop/PrestaShop/blob/2da8291bba521cf5af96454d3fd96e4cc2e7ba8d/js/tiny_mce/skins/prestashop/skin.min.css#L69-L71

My opinion would be removing these lines instead.

File: admin-dev/themes/new-theme/scss/components/_tinymce.sass (hunk 1 above)
Comment (dineshbadrukhiya) [reply admin-dev/themes/new-theme/scss/components/_tinymce.sass]: Hello @Quetzacoalt91 
I had already tried to modify `js/tiny_mce/skins/prestashop/skin.min.css` by removing line as you have suggested. But when I go to back-office, removed css style still there after clearing cache also. So I thinked that it's loads by default with js  :) and then tried to modify `admin-dev/themes/new-theme/scss/components/_tinymce.sass`.

I also dislike to use !important but here I need to override the style `.mce-btn:focus, .mce-btn:hover { 
     background-color: transparent !important; 
 } ` from `js/tiny_mce/skins/prestashop/skin.min.css` file. That's why I have used style with !imortant.

Comment (Quetzacoalt91) [reply admin-dev/themes/new-theme/scss/components/_tinymce.sass]: I managed to make it work on my side, did you make sure your browser cache was emptied?

Comment (dineshbadrukhiya) [reply admin-dev/themes/new-theme/scss/components/_tinymce.sass]: @Quetzacoalt91 I have cleared browser cache and it's fixed for me now.

Comment (dineshbadrukhiya) [reply admin-dev/themes/new-theme/scss/components/_tinymce.sass]: @Quetzacoalt91 Thanks. I would like to thank you @matks also as you helped me a lot on Gitter regarding this PR 😀

Comment (Quetzacoalt91) [reply admin-dev/themes/new-theme/scss/components/_tinymce.sass]: And now you can remove the changes applied in this file and `admin-dev/themes/new-theme/public/theme.css`, as they are properly applied on the popup correctly. :)

Comment (dineshbadrukhiya) [reply admin-dev/themes/new-theme/scss/components/_tinymce.sass]: @Quetzacoalt91  Does I need to remove old commits as I'm not good to remove it :).

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you @dineshbadrukhiya

**Pull request merged**